.storybook/main.js

```diff
@@ -1,12 +1,26 @@
+import { dirname, join } from "path";
 module.exports = {
   stories: [
     "../stories/**/*.stories.mdx",
     "../stories/**/*.stories.@(js|jsx|ts|tsx)",
   ],
+
   addons: [
-    "@storybook/addon-links",
-    "@storybook/addon-essentials",
-    "@storybook/addon-interactions",
+    getAbsolutePath("@storybook/addon-links"),
```

Comment:
I removed all the `getAbsolutePath` wraps and it still worked. Why do we need it?